src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViewsGrid.css

```diff
@@ -2,127 +2,445 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
+:root .grid-stack-item > .ui-resizable-handle {
```

Comment:
Why do you have :root here?

Reply:
ping. And it would be good to scope this down to items under .nb-grid-stack too

Reply:
Scoped it down. I was overriding a style from the library so I copied it as they had it there, that's why I added the :root.